### PR TITLE
docs: fix typo in README.md and docs related to environments supported

### PR DESCRIPTION
**Pull request by ThisIsMissEm**
Tiny follow up to #1820 and #1827 — note to self: acceptance criteria should really have the correct text in there in the future.

### CODE REVIEW

File: docs/api/source/index.rst [hunk 1]
```diff
@@ -39,16 +39,16 @@ ship both `ESM <https://nodejs.org/docs/latest-v16.x/api/esm.html>`__ and
 `CommonJS <https://nodejs.org/docs/latest-v16.x/api/modules.html>`__, with type
 definitions for TypeScript alongside.
 
-This mean that we only support environments (browsers or runtimes) that were
-released after mid-2018 out of the box, if you wish to target these
-environments, then you will need to cross-compile our SDKs via the use of
-`Babel <https://babeljs.io>`__, `webpack <https://webpack.js.org/>`__,
-`SWC <https://swc.rs/>`__, or similar.
+This means that out of the box, we only support environments (browsers or
+runtimes) that were released after mid-2018, if you wish to target other (older)
```
Comment (chelseapinka) [inline docs/api/source/index.rst:43]: I would almost just say "older" instead of but other and older

**Review by chelseapinka — APPROVED** [review]

File: docs/api/source/index.rst (hunk 1 above)
Comment (ThisIsMissEm) [reply docs/api/source/index.rst:43]: mm, I wanted to emphasise that's it's anything that's not this, not just "older" environments. Maybe `(e.g., older)` ?

Comment (chelseapinka) [reply docs/api/source/index.rst:43]: but what other environments other than the older ones actually apply?

Comment (ThisIsMissEm) [reply docs/api/source/index.rst:43]: I'm not sure of top of head, tbh. I just wanted to be more broad to cover all basis.

**Pull request merged**